# Bazaar revision bundle v0.8
#
# message:
#   round 3 of harring vs distutils; forego trying to use their options for setting compiler/flags, just force it directly from env access.
#   
# committer: Brian Harring <ferringb@gentoo.org>
# date: Mon 2006-10-02 21:57:25.611000061 -0700

=== modified file setup.py
--- setup.py
+++ setup.py
@@ -74,19 +74,8 @@
     def run(self):
         compiler = ccompiler.new_compiler(
             compiler=self.compiler, dry_run=self.dry_run, force=self.force)
-        sysconfig.customize_compiler(compiler)
-        cc = ' '.join(compiler.compiler)
-        
-        for x in ("BASECFLAGS", "CCSHARED", "LDFLAGS"):
-            f = sysconfig.get_config_var(x)
-            if isinstance(f, basestring):
-                cc = cc.replace(f, '')
-            elif f is None:
-                continue
-            else:
-                cc = cc.replace(" ".join(f), '')
-
-        compiler.set_executables(compiler=cc, compiler_so=cc)
+        cc = "%s %s" % (os.environ.get("CC", "cc"), os.environ.get("CFLAGS", ""))
+        compiler.set_executables(compiler=cc, compiler_so=cc, linker_exe=cc)
         objects = compiler.compile(list(
                 os.path.join('src', 'filter-env', name)
                 for name in ('main.c', 'bmh_search.c')), debug=self.debug)

# revision id: ferringb@gentoo.org-20061003045725-6fd8cb64a7e2d92d
# sha1: ffca209917036154225320e5990ecc0f6f50b16d
# inventory sha1: 80c748789eaf666c4b958c96100f3efd1004d4eb
# parent ids:
#   ferringb@gentoo.org-20061003002842-9919f5fcf9ae85f2
# base id: ferringb@gentoo.org-20061003002842-9919f5fcf9ae85f2
# properties:
#   branch-nick: integration

